src/chainparams.cpp

```diff
@@ -705,7 +640,7 @@ class CTestNetParams : public CChainParams {
  */
 class CDevNetParams : public CChainParams {
 public:
-    CDevNetParams(bool fHelpOnly = false) {
+    CDevNetParams(const ArgsManager& args) {
```

Comment:
@UdjinM6 should this be marked explicit?

```suggestion
    explicit CDevNetParams(const ArgsManager& args) {
```

Reply:
Good catch!